tools/depends/configure.ac

```diff
@@ -259,23 +244,25 @@ case $host in
     esac
     case $host in
       *86*-apple-darwin)
-        MC_CHECK_NOT_CPU([$use_cpu], "arm")
+        MC_CHECK_NOT_CPU([$use_arch], "arm")
```

Comment:
cpu should be never be arm[xxx], if you want to have a more specific architecture for arm (e.g. armv7-a) it has to be passed using --with-arch=armv7-a. CPU is something like cortex-A53 or xeon